resources/knative-build/values.yaml

```diff
@@ -5,5 +5,5 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   test_knative_build_acceptance:
-    dir: pr/
-    version: PR-4606
+    dir: develop/
```

Comment:
Hi Rakesh,
could you please mention this image bump in the proposed changes for this PR as well and map it to the component change in the PR, so that we have reference to the actual change as well.

Reply:
done.